<?xml version="1.0"?>

<launch>

  <node pkg="traj_generator" type="traj_test" name="traj_test" output="screen">
    <param name="follow_yaw" type="bool" value="false"/>
    <param name="mean_speed" type="double" value="0.8"/>

    <!--remap from="~odom" to="/pos_vel_mocap/odom_TA" /-->

    <!-- <remap from="~odom" to="/vins_multi_rgbd/imu_propagate" /> -->

    <remap from="~odom" to="/ekf/ekf_odom" />

  </node>

</launch>
